src/hints/mod.rs

```diff
@@ -336,3 +339,26 @@ pub fn breakpoint(
     println!("\n-----------END BREAKPOINT-----------\n");
     Ok(())
 }
+
+#[allow(unused)]
```

Comment:
Can't you add the mapping in the hints map?

Reply:
I was confused at first how #4 managed to not need `unused` and also didn't add it to the map, it's because it's `pub` in that case.

Reminder: #4 needs to include this in the hint map too :)

Reply:
Fixed in 63bb84d